coalib/output/ConsoleInteraction.py

```diff
@@ -201,23 +214,18 @@ def print_lines(console_printer,
 
         printed_chars = 0
         if i == sourcerange.start.line and sourcerange.start.column:
-            console_printer.print(highlight_text(
-                no_color, line[:sourcerange.start.column - 1], lexer), end='')
+            print_normal(line[:sourcerange.start.column - 1])
```

Comment:
BackgroundSourceRangeStyle was not in the old call but in the new one.
So maybe this also is a print_normal?